File: java/Processor.java

class Processor {
    private String name;
    private int price;

    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    public Processor() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public void printProcessor() {
        System.out.println("Processor Name: " + this.name);
        System.out.println("Processor Price: " + this.price);
    }
}

File: php/Processor.php
<?php
class Processor
{
    private $name;
    private $price;

    public function __construct($name = "", $price = 0)
    {
        $this->name = $name;
        $this->price = $price;
    }

    public function getName()
    {
        return $this->name;
    }

    public function setName($name)
    {
        $this->name = $name;
    }

    public function getPrice()
    {
        return $this->price;
    }

    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function printProcessor()
    {
        echo "Processor Name: " . $this->name . "<br/>";
        echo "Processor Price: " . $this->price . "<br/>";
    }
}
